lib/core/src/packStruct.cpp

```diff
@@ -2918,16 +2706,16 @@ unpackPointerItem( packItem_t *myPackedItem, void **inPtr,
         if ( myDim == 0 ) {
             outPtr = addPointerToPackedOut( unpackedOutput,
                                             numElement * elementSz, NULL );
-            status = unpackIntToOutPtr( inPtr, &outPtr, numElement,
-                                        myPackedItem, irodsProt );
+            status = unpackIntToOutPtr( inPtr, outPtr, numElement,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)